int32 ID
int32 pos
int32 speed
int32 force
int32 maxTime
